docs/content/preview/explore/ysql-language-features/pg-extensions.md

```diff
@@ -63,7 +63,7 @@ Currently, in a multi-node setup, you need to install the extension on _every_ n
 
 In a read replica setup, install extensions on the primary instance, not on the read replica. Once installed, the extension replicates to the read replica.
 
-You cannot install new extensions in YugabyteDB Managed. If you need a database extension that is not pre-bundled with YugabyteDB added to a YugabyteDB Managed cluster, contact {{<support-cloud>}} with the names of the cluster and extension, or [reach out on Slack](https://yugabyte-db.slack.com/).
+You cannot install new extensions in YugabyteDB Managed. If you need a database extension that is not pre-bundled with YugabyteDB added to a YugabyteDB Managed cluster, contact {{%support-cloud%}} with the names of the cluster and extension, or [reach out on Slack](https://yugabyte-db.slack.com/).
```

Comment:
It's easier to read (and search) if you space the tag away from the delimiters... not required by Hugo, but the % in particular makes the tag names harder to read... (Also, there's a slack invitation shortcode too, if that makes more sense than the direct slack link? not sure...)
```suggestion
You cannot install new extensions in YugabyteDB Managed. If you need a database extension that is not pre-bundled with YugabyteDB added to a YugabyteDB Managed cluster, contact {{% support-cloud %}} with the names of the cluster and extension, or [reach out on Slack](https://yugabyte-db.slack.com/).
```